en/docs/guides/identity-federation/google.md

```diff
@@ -152,13 +151,68 @@ For example. `http://localhost.com:8080/pickup-dispatch`
 
     ![Pickup-dispatch application]({{base_path}}/assets/img/samples/pickup-dispatch-login.png)
 
-
 2. Click **Login**. You are redirected to the Google login page.
   
     ![Google login page]({{base_path}}/assets/img/samples/sign-in-google.png)
-    
+
 3. Sign in using your Google credentials. You are redirected to the
     Pickup sample homepage.
 
 !!! info "Related topics" 
-    - [Concepts: Introduction to Identity Federation]({{base_path}}/references/concepts/identity-federation/)
\ No newline at end of file
+    - [Concepts: Introduction to Identity Federation]({{base_path}}/references/concepts/identity-federation/)
+
+## Google One Tap
+Google One Tap is a personalized authentication feature provided by Google as a trending federated authentication option. When a browser has an authenticated google session, a personalized sign in/sign up button will appear instead of a conventional Google sign in button for One-Tap supported web applications.
```

Comment:
```suggestion
Google One Tap is a personalized authentication feature provided by Google as a trending federated authentication option. When a browser has an authenticated google session, a personalized sign-in/sign-up button will appear instead of a conventional Google sign-in button. Note that this feature applies to One Tap-supported web applications.
```